src/citrine/_session.py

```diff
@@ -147,6 +147,7 @@ def checked_request(self, method: str, path: str,
 
         if self._is_access_token_expired():
             self._refresh_access_token()
+            self._check_accounts_version()
```

Comment:
I wouldn't expect this to change very often, but this is controlled by a feature flag which could get toggled at any point.  It's probably good to check this on refresh to make sure we have the right setting.

Reply:
I also have it here in case someone has an open script that they are in the middle of running (which happens sometimes because of all the long running tasks) when the switch over occurs.